presto-docs/src/main/sphinx/functions/tdigest.rst

```diff
@@ -0,0 +1,41 @@
+=========================
+T-Digest Functions
+=========================
+
+Data Structures
+---------------
+
+A T-digest is a data sketch which stores approximate percentile
+information.  The Presto type for this data structure is called ``tdigest``.
+T-digests may be merged, and for storage and retrieval they may be cast
+to/from ``VARBINARY``.
+
+Functions
+---------
+
+.. function:: merge(tdigest) -> tdigest
+    :noindex:
+
+    Aggregation function. Merges all input ``tdigest``\ s into a single ``tdigest``.
+
+.. function:: value_at_quantile(tdigest, quantile) -> double
+    :noindex:
+
+    Returns the approximate percentile value from the T-digest given
+    the number ``quantile`` between 0 and 1.
+
+.. function:: values_at_quantiles(tdigest, quantiles) -> array<double>
+    :noindex:
+
+    Returns the approximate percentile values as an array given the input
+    T-digest and array of values between 0 and 1 which
+    represent the quantiles to return.
+
+.. function:: tdigest_agg(x) -> tdigest
+
+    Returns the ``tdigest`` which is composed of  all input values of ``x``.
+
+.. function:: tdigest_agg(x, w) -> tdigest
+
+    Returns the ``tdigest`` which is composed of  all input values of ``x`` using
```

Comment:
Similar to above and explain x and w

Reply:
It says `per-item weight w`, so I consider `w` explained. I added a note that `w` must be >= 1.

Reply:
any < restriction for w?

Reply:
No upper bound restriction.
I added information that `x` and `w` can be of any numeric type.